changelogs/unreleased/gh-3211-per-module-log_level.md

```diff
@@ -0,0 +1,8 @@
+## feature/box
+
+* Added new `print_module_name` option to `log.cfg{}` and `box.cfg{}` that
+  enables printing in the log the name of a module from which the logging
+  function was called (gh-3211).
+
+* Supported per-module log level setting via `log.cfg{level_modules = {...}}`
+  or `box.cfg{log_level_modules = {...}}` (gh-3211).
```

Comment:
I guess we also need to mention `log_level_modules` in document request and  how the module logger get automatic name.

Reply:
1. `log_level_modules` is already mentioned there.
2. Done.